binding/Binding/SKPaint.cs

```diff
@@ -248,6 +248,35 @@ public long BreakText (IntPtr buffer, IntPtr length, float maxWidth, out float m
 			return (long)SkiaApi.sk_paint_break_text (Handle, buffer, length, maxWidth, out measuredWidth);
 		}
 
+		public SKPath GetTextPath(string text, float x, float y)
+		{
+			if (text == null)
+				throw new ArgumentNullException(nameof(text));
+			return new SKPath(SkiaApi.sk_paint_get_utf16_text_path(Handle, text, (IntPtr)text.Length, x, y));
+		}
+
+		public SKPath GetTextPath(IntPtr buffer, IntPtr length, float x, float y)
+		{
```

Comment:
I did this to be similar to the other APIs that handled text. 

Peter

> On Apr 8, 2016, at 7:43 AM, Bill Holmes <notifications@github.com> wrote:
> 
> In binding/Binding/SKPaint.cs:
> 
> > @@ -248,6 +248,35 @@ public long BreakText (IntPtr buffer, IntPtr length, float maxWidth, out float m
> >  			return (long)SkiaApi.sk_paint_break_text (Handle, buffer, length, maxWidth, out measuredWidth);
> >  		}
> >  
> > +		public SKPath GetTextPath(string text, float x, float y)
> > +		{
> > +			if (text == null)
> > +				throw new ArgumentNullException(nameof(text));
> > +			return new SKPath(SkiaApi.sk_paint_get_utf16_text_path(Handle, text, (IntPtr)text.Length, x, y));
> > +		}
> > +
> > +		public SKPath GetTextPath(IntPtr buffer, IntPtr length, float x, float y)
> > +		{
> Do you need the IntPtr version of these? It seems that the string version would be all that we need.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>